cocos/audio/android/AudioPlayerProvider.cpp

```diff
@@ -71,7 +71,7 @@ struct AudioFileIndicator
 static AudioFileIndicator __audioFileIndicator[] = {
         {"default", 128000}, // If we could not handle the audio format, return default value, the position should be first.
         {".wav",    1024000},
-        {".ogg",    128000},
+        {".ogg",    512000}, //128000
```

Comment:
Ogg file with 512K is little big. It normally represents a background music.
I know you wanna use PcmAudioPlayer for your background music since UrlAudioPlayer will cause gap while looping on some android devices.
But I suggest to revert this line since no all developers have a music less than 512K.

Reply:
@piotrros , could you please review my comment above?